gaia_spatialstats/processes.py

```diff
@@ -198,8 +198,143 @@ def compute(self):
         self.output.write()
         logger.debug(self.output)
 
+
+class GearyCProcess(GaiaProcess):
+    u"""
```

Comment:
I've removed the u. One of the apostrophe in there was a unicode character but that has been removed.